apps/macos/package-lock.json

```diff
@@ -0,0 +1,2013 @@
+{
```

Comment:
same for this file

Reply:
Removing this - not sure why it was added :)